src/store/wizardSlice.ts

```diff
@@ -45,6 +46,9 @@ type wizardState = {
   openScap: {
     profile: DistributionProfileItem | undefined;
   };
+  repositories: {
+    customRepositories: CustomRepository[] | undefined;
```

Comment:
If we are going to initialize `customRepositories` as `[]` and use that as the empty state, do we need to allow the `undefined` type? Kind of similar to how our controlled inputs are just `''` for the empty state and only strings are allowed.

Reply:
Good point, I've removed the `undefined`